frontend/src/pages/GraphPF/GraphPFElems.tsx

```diff
@@ -451,7 +457,7 @@ const getEdgeLabel = (edge: EdgeModel, nodeMap: NodeMap, settings: GraphPFSettin
     }
   }
 
-  let label = labels.join('\n');
+  let label = labels.join(' | ');
```

Comment:
this is a decent alternative.

Note that the cytoscape graph doesn't have the double-labels for the bidirectional edges.  I guess this is OK, we are starting to diverge between the two graph impls.  I also noticed a highlighting issue in the cyto graph, related to waypoints (not a crash).

Reply:
Right, I just focused this changes in the PF graph. 

Regarding the highlighting issue, I think this is because now in PF the edge is not pushed, so it is easier to follow the path. But when there are two edges, there are more related nodes, and sometimes all the graph is highlighted, which might be confusing.

Reply:
I'm not going to worry too much about the Cy highlighting issue.